source/frontend/src/features/properties/map/acquisition/update/summary/UpdateAcquisitionForm.tsx

```diff
@@ -69,6 +73,41 @@ export const UpdateAcquisitionForm = React.forwardRef<
               </SectionField>
             </Section>
 
+            <Section header="Project">
+              <SectionField label="Ministry project">
+                <AsyncTypeahead
+                  field="project"
+                  labelKey="text"
+                  isLoading={isTypeaheadLoading}
+                  options={matchedProjects}
+                  onSearch={handleTypeaheadSearch}
+                />
+              </SectionField>
+              <SectionField label="Product">
+                <Select field="product" options={[]} placeholder="Select..." />
+              </SectionField>
+              <SectionField label="Funding">
+                <Select
+                  field="fundingTypeCode"
+                  options={acquisitionFundingTypes}
+                  placeholder="Select..."
+                  onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+                    const selectedValue = [].slice
```

Comment:
had to look this one up but seems fine.

Reply:
Yeh, rather convoluted way of updating the value